src/Roslyn.Suppressions/Roslyn.Suppressions/InProcess/RoslynSettingsFileSynchronizer.cs

```diff
@@ -68,24 +69,24 @@ internal sealed class RoslynSettingsFileSynchronizer : IRoslynSettingsFileSynchr
         {
         }
 
-        internal RoslynSettingsFileSynchronizer(ISuppressedIssuesMonitor suppressedIssuesMonitor,
+        internal RoslynSettingsFileSynchronizer(IServerIssuesStore serverIssuesStore,
             ISonarQubeIssuesProvider suppressedIssuesProvider,
             IRoslynSettingsFileStorage roslynSettingsFileStorage,
             IActiveSolutionBoundTracker activeSolutionBoundTracker,
             ILogger logger,
             IThreadHandling threadHandling)
         {
-            this.suppressedIssuesMonitor = suppressedIssuesMonitor;
+            this.serverIssuesStore = serverIssuesStore;
             this.suppressedIssuesProvider = suppressedIssuesProvider;
             this.roslynSettingsFileStorage = roslynSettingsFileStorage;
             this.activeSolutionBoundTracker = activeSolutionBoundTracker;
             this.logger = logger;
             this.threadHandling = threadHandling;
 
-            suppressedIssuesMonitor.ServerSuppressionsChanged += SuppressedIssuesMonitor_SuppressionsUpdateRequested;
+            serverIssuesStore.ServerIssuesChanged += ServerIssuesChanged;
         }
 
-        private void SuppressedIssuesMonitor_SuppressionsUpdateRequested(object sender, EventArgs e)
+        private void ServerIssuesChanged(object sender, EventArgs e)
```

Comment:
We don't have a consistent naming pattern for event handlers, but two common ones are "OnXXX" and "HandleXXX" e.g. `OnServerIssuesChanged` and `HandleServerIssuesChanged`.

Using exactly the same name as the event can be a bit confusing e.g. if you do a text search, it might not be obvious whether you are looking at the event or the handler.